tools/kebab-migration.toit

```diff
@@ -0,0 +1,168 @@
+import cli
```

Comment:
Add the correct copyright and license information.

Reply:
done.